rethinkdb_exporter.go

```diff
@@ -240,15 +262,9 @@ func extractAllMetrics(sess *r.Session, scrapes chan<- scrapeResult) error {
 				if !*countRows || !*getTableStats {
 					continue
 				}
-				res, err := r.DB(s.DB).Table(s.Table).Count().Run(sess)
-				if err != nil {
-					return err
-				}
-				var count float64
-				if err = res.One(&count); err != nil {
-					return err
-				}
-				scrapes <- s.newScrapeResult("table_docs_total", count)
+
+				wg.Add(1)
+				go countTableDocs(s.Server, s.DB, s.Table, sess, scrapes, wg)
```

Comment:
I'd suggest you use https://godoc.org/golang.org/x/sync/errgroup to not ignore the errors that might happen inside the call to `countTableDocs()`.

Reply:
done